app/data_mapper_setup.rb

```diff
@@ -0,0 +1,9 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+require_relative 'models/message'
+require_relative 'models/user'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}" )
+DataMapper.finalize
+DataMapper.auto_upgrade!
```

Comment:
Consider extracting this to an individual rake file - https://github.com/makersacademy/chitter-challenge/blob/master/docs/review.md#ensure-rakefile-has-appropriate-tasks